src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[new Random().nextInt(10)];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (Figure test : figures) {

```

Comment:
```suggestion
        for (Figure figure : figures) {

```

don't forget about variables naming